composer.json

```diff
@@ -88,7 +88,6 @@
             "Rector\\Utils\\": "utils"
         },
         "files": [
-            "src/functions/node_helper.php",
```

Comment:
`src/functions` can be removed from downgrade workflow

https://github.com/rectorphp/rector-src/blob/4765d5a18552704431a5064a308b73a2414c2190/.github/workflows/build_scoped_rector.yaml#L55

Reply:
Done in main :+1: